### ISSUE TITLE
TextBook vs Scheduled Topic

### ISSUE THREAD

**Issue by albertsutz** [nus-cs2103-AY2122S2/forum#300]
Hi Prof!

If I wanted to review for the final exams, is reading the TextBook and the Scheduled Topic exactly the same? (I know that the scheduled topic is from the textbook, just want to make sure that it has covered everything).

Thanks in Advance!

**Comment by damithc** [comment]
@albertsutz Yes, they should be the same. Textbook (printer-friendly version) is recommended as it omits optional topics and contains all examinable topics.
